test/role.test.js

```diff
@@ -612,19 +612,20 @@ describe('role model', function() {
           var uniqueRoleName = 'testRoleFor' + principalType;
           Role.create({name: uniqueRoleName}, function(err, role) {
             if (err) return done(err);
+
             role.principals.create({principalType: principalType, principalId: model.id},
-            function(err, p) {
-              if (err) return done(err);
-              var pluralName = Model.pluralModelName.toLowerCase();
-              role[pluralName](function(err, models) {
+              function(err, p) {
```

Comment:
Changes in this file look like they are only changing indentation/whitespace, can we revert please?